### ISSUE TITLE
Possible to have native LSP attached to the acwrite buffer?

### ISSUE THREAD

**Issue by hrmJ** [hoschi/yode-nvim#12]
Thanks for the awesome plugin! I was just wondering, if anyone has had any luck with having native lsp clients attached to the acwrite buffers? It seems like the LSP features are not working and `LspInfo` shows 0 of x clients attached to the buffer when doing a yode-nvim split / float.

**Comment by hoschi** [comment]
I had no problems with coc to scan the code in the buffer, but didn't tried with native LSP. It needs deeper integration to resolve the location in the yode buffer to the correct position in the real buffer to have all LSP features working.

**Comment by hrmJ** [comment]
Right, probably a bit too involved for me to try to set it up. Glad to know it works with coc, though
